pydantic/networks.py

```diff
@@ -162,7 +427,7 @@ class AnyUrl(_BaseUrl):
     @property
     def host(self) -> str:
         """The required URL host."""
-        ...
+        return self._url.host  # type: ignore
```

Comment:
Maybe @Viicos has a better idea here - the point is, the base `Url` doesn't need a host, but this more specific implementation does, so here's how I'm keying in on the `host` as a `str`...